### PR TITLE
added notes to tech orientation guide

### PR DESCRIPTION
**Pull request by mdewey**
- formated with markdown prettier
- added TOC
- updated and refactored the front sections
- added FAQ
- updated and refactored software tools

### CODE REVIEW

File: teams/vsa/onboarding/VSA Technical Orientation.md [hunk 1]
```diff
@@ -1,354 +1,489 @@
 # VSA Technical Orientation
+
 Technical orientation from the VSA Technical Lead ([pshort@governmentcio.com](mailto:pshort@governmentcio.com)/[paul.short@va.gov](mailto:paul.short@va.gov)) as of 05/05/2020. Primary audience are VSA front end and back end developers, but other VSA team members may find some of the links here useful. There is a lot of decent documentation available; however, some of it is changing, redundant, outdated, or requires more context in order to navigate. This is an attempt to capture the most important links in a structured order for a new developer.
 
+## Table of Contents
```
Comment (short000) [inline teams/vsa/onboarding/VSA Technical Orientation.md:5]: A TOC is a great idea. Are you using a VS Code Plugin to help generate these from ##, or is this manual for now?

**Comment by mdewey** [comment]
I created it manually, but the markdown extension I have keeps it up to date

File: teams/vsa/onboarding/VSA Technical Orientation.md [hunk 2]
```diff
@@ -1,354 +1,489 @@
...
+| Chrome&nbsp;Extension: _Lighthouse_              | [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)                                                            | [508 accessibility scanning](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#getting-started-with-automation): _recommended_. Audits for web performance, accessibility, progressive web apps, SEO and more.                                                                                        |
+
+### (optional) VS Code Extensions
```
Comment (short000) [inline teams/vsa/onboarding/VSA Technical Orientation.md:351]: glad you added this, was on my to-do list. I ran a survey before and see that you've captured the big ones but also good ones we missed

File: teams/vsa/onboarding/VSA Technical Orientation.md [hunk 3]
```diff
@@ -1,354 +1,489 @@
...
-
-*   **MilitaryCAC<span/>.com** \
-VA PIVs are essentially the same (hardware and driver-wise) as the DOD Common Access Cards (CACs), so I’ve found https://militarycac.com to be helpful if you have to get down to device/driver troubleshooting details. The site isn’t officially endorsed by the DOD or the VA, but is a community-recognized, grass-roots effort to fill in the gaps left by government help desks. Some of the context is Army/DOD-specific, but if you are technical enough or have the time to sort through those differences, the advice can be applied to VA PIV readers as well.
-    *   https://militarycac.com/macnotes.htm#Mac_friendly
-    *   https://militarycac.com/usbreaders.htm
+
+- Bookmark the VA Citrix Remote Access login site: https://citrixaccess.va.gov/
+- VA Remote Access Help: https://www.oit.va.gov/resources/remote-access/
+- VA CAG Remote Access Connectivity Video Walkthrough: https://www.youtube.com/watch?v=073hayTIiwA
+- Initial setup instructions for VA email and CAG access should come to you via email. If you haven’t gotten yours, check with our onboarding coordinator, Julie Ramirez ([Julie.Ramirez2@va.gov](mailto:Julie.Ramirez2@va.gov))
```
Comment (short000) [inline teams/vsa/onboarding/VSA Technical Orientation.md:417]: I'll need to change the onboarding coordinator, but will do it after the review

File: teams/vsa/onboarding/VSA Technical Orientation.md [hunk 4]
```diff
@@ -1,354 +1,489 @@
...
+
+> Where can find mock data?
+
+There are two repositories. There is there the [mock data for the API](https://github.com/department-of-veterans-affairs/vets-api-mockdata) and then there is the [fake log in information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/3a1b22e261185ef42f3f7f849024b759bcfa9dad/Administrative/vagov-users/staging-test-accounts.md)
+
+> What is the mission of this project?
+
+Check out this repo and read through it: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/content. This really defines the voice and mission of the project and how we as tech team are trying to approach it.
+
+> I found something in this guide that was wrong/missing/could use improvement?
```
Comment (short000) [inline teams/vsa/onboarding/VSA Technical Orientation.md:487]: good way to end the FAQ and whole document; let's be sure that future edits keep this the last item

File: teams/vsa/onboarding/VSA Technical Orientation.md [hunk 5]
```diff
@@ -1,354 +1,489 @@
...
+- **MilitaryCAC<span/>.com** \
+  VA PIVs are essentially the same (hardware and driver-wise) as the DOD Common Access Cards (CACs), so I’ve found https://militarycac.com to be helpful if you have to get down to device/driver troubleshooting details. The site isn’t officially endorsed by the DOD or the VA, but is a community-recognized, grass-roots effort to fill in the gaps left by government help desks. Some of the context is Army/DOD-specific, but if you are technical enough or have the time to sort through those differences, the advice can be applied to VA PIV readers as well.
+  _ https://militarycac.com/macnotes.htm#Mac_friendly
+  _ https://militarycac.com/usbreaders.htm
 
 ## VA Service Desk
+
 The VA National Service Desk (NSD) (aka VA Enterprise Service Desk) number is 855-NSD-HELP (855-673-4357). For this project you might use it for issues with VA Citrix Access Gateway (CAG), VA email, VA Talent Management System (TMS), or outages/downtime associated with VA Enterprise Services.
+
+## Onboarding FAQs
```
Comment (short000) [inline teams/vsa/onboarding/VSA Technical Orientation.md:449]: This is great

File: teams/vsa/onboarding/VSA Technical Orientation.md [hunk 6]
```diff
@@ -1,354 +1,489 @@
...
+
+Checkout out the VA getting start [site](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/). There are two sections on the left, `VA.gov workflow` and `Common tasks`. read through those two sections and try to implement some ideas in a branch.
+
+> Where do I go with my initial technical questions about the site?
+
+First, go to your onboarding buddies, then [slack channels](#slack) that seem appropriate. Ask questions, this the time to really see what is going on.
+
+> Do we have a style guide?
+
+We use prettier and eslint to enforce the Airbnb style guide. We have customized it a little, but usually you editor should be configured to automatically apply those stylings.
```
Comment (short000) [inline teams/vsa/onboarding/VSA Technical Orientation.md:469]: Would like some more detail on "customized it a little". maybe this is missing in VSP's documentation. For now it's good to at least mention it.

**Review by short000 — APPROVED** [review]
this is great. some comments, but will merge this in right away

**Pull request merged**